src/main/java/Bellroy/Ui.java

```diff
@@ -0,0 +1,112 @@
+package Bellroy;
+
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * The Ui class handles reading inputs from the user and printing the output to the console
+ */
+public class Ui {
+    private Scanner scanner = new Scanner(System.in);
+
+    /**
+     * prints the initial message to the user.
+     */
+    public void printWelcomeMessage() {
+        System.out.println("____________________________________________________________\n" +
```

Comment:
Perhaps these repeated lines of System.out.println("_________") can be put into a function called printLine()? Would be easier if you have to make modifications to it :)